charts/kong-operator/values.yaml

```diff
@@ -99,3 +99,7 @@ certsDir:
 # Override the default deployment selector labels. This is useful if you don't want
 # to use the defaults or are migrating to this chart and want to use existing labels.
 # selectorLabels: []
+
+# enable configuration dump for control planes.
+enableControlplaneConfigDump: false
+controlplaneConfigDumpPort: 10256
```

Comment:
This feature is controller via the added CLI flag / env so this option could conflict with with those being set to different value.

So to avoid the conflict I think we should configure the options of config dump based on the CLI flags/envs. WDYT?

Reply:
The variables are also used for generating `Service`s to access the config dump server. It would be more complex to extract the ports when we extract the options from the `env` .

Reply:
I agree that it would be more complicated but we have to handle the conflicts with CLI/env somehow. Perhaps we fail with a message when both are configured?

Reply:
Added a check on envs configured by other variables and fails with a message if they are set in env.